docs/version0.18.md

```diff
@@ -47,6 +48,10 @@ You can set the `-Xshareclasses:noTimestampChecks` option to turn off timestamp
 
 On z/OS 64-bit, [`-Xmso`](xmso.md) has a 1 MB minimum value, to match the minimum stack space provided by the operating system. If you set a value smaller than 1 MB, the value is ignored.
 
+### -XX:+TransparentHugePage is enabled by default on Linux&reg POWER&reg and IBM Z&reg systems
+
+On Linux systems for POWER and IBM Z, [-XX:+TransparentHugePage](xxtransparenthugepage.md) is enabled by default. This option only takes affect when Transparent Huge Pages (THP) is set to `madvise` on your system. When transparent huge pages are used, application footprint may increase.
```

Comment:
Misplaced "only": This option takes affect only when .....
Replace "may" with "might".

Reply:
Fixed.